include/psp2/kernel/clib.h

```diff
@@ -14,6 +14,21 @@
 extern "C" {
 #endif
 
+/**
+ * A handle for a managed memory space.
+ */
+typedef void* SceClibMspace;
+
+/**
+ * Structure for stats of SceClibMspace
+ */
+typedef struct SceClibMspaceStats {
+    SceSize capacity;     //!< Capacity of the Mspace
+    SceSize unk;          //!< Unknown, value is equal to capacity
+    SceSize peakInUse;    //!< Peak memory allocated
```

Comment:
can you use `snake_case` for the field name?
```suggestion
    SceSize peak_in_use;    //!< Peak memory allocated
```